modules/libcom/src/iocsh/iocsh.cpp

```diff
@@ -1367,6 +1368,35 @@ iocshEnvClear(const char *name)
     }
 }
 
+void epicsStdCall
+iocshSet(const char *name, const char * value)
```

Comment:
The new functions `iocshSet()` and `iocshShow()` should be marked `static` and not `epicsStdCall`; they will only be called from within this file. In fact unless they are called from elsewhere in this file (I don't think they are) it would be better to combine their functionality into the `iocshSetCallFunc()` and `iocshShowCallFunc()` routines directly; there's no need for them to even be separate routines.